source/documentation/other-topics/migrate-to-live-sa-github-actions.html.md.erb

```diff
@@ -0,0 +1,64 @@
+---
+title: Migrating serviceaccount module for GitHub actions to the "Live" cluster
+last_reviewed_on: 2021-09-17
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+## Migrating service account module
+
+
+#### Skip file to your namespace in live-1
+
+Make sure you have [skip your namespace][long-running] file in live-1, this will allow the Apply Pipeline to skip
```

Comment:
This is added when the users do the migrate using cli. Can we say, make sure you run the `cloud-platform environment migrate` instead?